CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ project(tritonclient LANGUAGES C CXX)
 # Options
 #
 set(TRITON_VERSION "0.0.0" CACHE STRING "Version for the clients")
+set(PERF_ANALYZER_VERSION "0.0.0" CACHE STRING "BUILD VERSION for Perf Analyzer")
```

Comment:
I would prefer to keep the terminology consistent with the TRITON_VERSION variable name.

Reply:
yes, keep it as is.

Reply:
Ok. I am concerned someone might get confused reading that the version is here but then looking at the constants file and seeing something else called version but it's a mixture of this version and the GIT SHA. Would be nice to have consistent and distinct names for the two "versions".

Reply:
They have 2 different variable names. It is unclear where the confusion comes in. Can you explain further please?

Reply:
The variable from the line of code from this thread is called `PERF_ANALYZER_VERSION`. We know it's gonna be set to the release (e.g. `23.06`).

The variable in `constants.h` is called `VERSION`. We can reasonably assume that since it's a variable called `VERSION` inside the Perf Analyzer repository `constants.h` file, that it is referring to the _Perf Analyzer Version_. We know it is going to be set to the release plus the git sha (e.g. `23.06 (commit 7d5dd7b`).

Therefore, we have two variables that both represent the _Perf Analyzer Version_ and yet have different values.

I can imagine someone new to the repo reading this and wondering "which one is the real perf analyzer version...?"

Reply:
to be clear, this will never be 23.06. It will be 2.33 or something.  version != release 
I don't like the RELEASE variable for this reason.

seems like SHA and VERSION should be in constants.h, as they are actually constants. The function should be the one that stitches them together for printing.

Reply:
done

Reply:
I misunderstood. Tim's right. This won't hold a release value. All good here.